static/app/views/issueList/customViewsHeader.tsx

```diff
@@ -289,7 +289,7 @@ function CustomViewsIssueListHeaderTabsContent({
       return;
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [navigate, organization.slug, query, sort, viewId]);
+  }, [navigate, organization.slug, query, sort, viewId, tabListState]);
```

Comment:
This is a good example of the eslint rule trying to help us, would be good to take a second look at some point and see if we can remove the ignores